src/Uno.UI.Runtime.Skia.Gtk/Extensions/Storage/Pickers/FileOpenPickerExtension.cs

```diff
@@ -0,0 +1,90 @@
+﻿#nullable enable
+
+using Gtk;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Runtime.Skia;
+using Windows.ApplicationModel.Resources;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+
+namespace Uno.Extensions.Storage.Pickers
+{
+	internal class FileOpenPickerExtension : IFileOpenPickerExtension
+	{
+		private readonly FileOpenPicker _picker;
+
+		public FileOpenPickerExtension(object owner)
```

Comment:
We now actually have a new way of registering extensions with strongly typed owner - 

https://github.com/unoplatform/uno/blob/68f39db6e7aaf3e76f8672b8e610a159513c428c/src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs#L56

The other extensions are not yet updated to this new approach, but for the new ones it would be nice :-) .

Reply:
@MartinZikmund Thanks. I've changed the registration to have the strongly typed owner.